builds/mingw32/Makefile.mingw32

```diff
@@ -40,10 +40,10 @@ all: libzmq.dll
 perf: inproc_lat.exe inproc_thr.exe local_lat.exe local_thr.exe remote_lat.exe remote_thr.exe
 
 libzmq.dll: $(OBJS)
-	g++ -shared -o $@ $^ -Wl,--out-implib,$@.a $(LIBS)
+	g++ -shared -static -O2 -s -o $@ $^ -Wl,--out-implib,$@.a $(LIBS)
```

Comment:
Why hardcoding O2 and s? Usually they are defined by the environment, application makefiles should never explicitly define them